highlighting/lib/src/language_definition_parts/callbacks.dart

```diff
@@ -1,8 +1,8 @@
-import 'package:highlighting/languages/common/mathematica_symbols.dart';
```

Comment:
1. Move everything from `language_definition_parts` directory to `src/`.
2. Move `mathematica_symbols.dart` to `src/languages`.